animation_nodes/nodes/mesh/mesh_object_input.py

```diff
@@ -33,7 +32,7 @@ def create(self):
         self.newOutput("Integer List", "Material Indices", "materialIndices")
 
         self.newOutput("Text", "Mesh Name", "meshName")
```

Comment:
Omar, Mesh name output has a bug.

Reply:
What is the bug?

Reply:
When I connect the viewer node to the mesh name output then it breaks the node-tree:
![Screenshot from 2019-12-23 15-45-23](https://user-images.githubusercontent.com/30294746/71352107-67cf0300-259b-11ea-8f68-62e43fd068fc.png)

Reply:
Fixed in b442dfe.

Reply:
Okay :)